TopPlaces/TopPlaces/Controllers/PhotosTableViewController.m

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "PhotosTableViewController.h"
+
+#import "DetailsPhotoViewController.h"
+#import "PlacesPhotosProvider.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@interface PhotosTableViewController()
+
+@end
+
+@implementation PhotosTableViewController
+#pragma mark - UIView overrides
+- (void) fetchPhotos {
+  //abstract
+  assert(0);
+}
+
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(nullable id)sender {
+  if ([segue.identifier isEqualToString:@"showPhoto"]) {
+    NSLog(@"prepareForSegue: showPhoto");
+    UINavigationController *nvc = (UINavigationController *)segue.destinationViewController;
```

Comment:
Avoid such abbreviations, navigator / navigationController and photoController would be better.